filer/private/sass/settings/_custom.scss

```diff
@@ -44,7 +44,7 @@ $padding-large: 20px;
 
 $screen-mobile: 320px;
 $screen-tablet: 720px;
-$screen-desktop: 1024px;
+$screen-desktop: 975px;
```

Comment:
maybe it's better to create new one breakpoint? Or it will fix all other places too?